pkg/resource/aws/aws_route.go

```diff
@@ -14,6 +14,7 @@ func initAwsRouteMetaData(resourceSchemaRepository resource.SchemaRepositoryInte
 	resourceSchemaRepository.SetNormalizeFunc(AwsRouteResourceType, func(res *resource.AbstractResource) {
 		val := res.Attrs
 		val.SafeDelete([]string{"timeouts"})
+		val.SafeDelete([]string{"carrier_gateway_id"})
```

Comment:
This field shouldn't be ignored totally, we only want to avoid diff from `nil` to `""` and vice-versa.

The method `DeleteIfDefault` is more appropriate

Reply:
Right, fix done.